assets/settings/default.json

```diff
@@ -279,6 +279,13 @@
   "relative_line_numbers": false,
   // If 'search_wrap' is disabled, search result do not wrap around the end of the file.
   "search_wrap": true,
+  // Search options to enable by default.
+  "search_defaults": {
```

Comment:
What if we call it `search`?
Then we could host more settings if anything comes up in the future.

Reply:
I was going to call it search_options but there's already a struct called SearchOptions. Search is a bit too wide. Maybe search_options and live with the internal and external name not matching!?

Reply:
I am still into just `search` as a key in that json, same as we have `git`, `terminal`, `gutter`, etc.
`"search"` seems to be vacant?